src/container.h

```diff
@@ -92,4 +92,53 @@ class RingDeque {
   T buffer_[CAPACITY];
 };
 
+//Stupid reimplementation of the deque
```

Comment:
Whenever you use comments in this code, please add a space after the // (also many other places in this pull request).

Regarding this TODO: is this something you intend to address, or is this indeed a class that you want to keep, or want to use the RingDeque ? In any case, it might need an update in the comment.